guides/source/6_0_release_notes.md

```diff
@@ -211,10 +211,240 @@ Please refer to the [Changelog][active-record] for detailed changes.
+    ([Commit](https://github.com/rails/rails/commit/f59b08119bc0c01a00561d38279b124abc82561b))
+
+*   Deprecate passing `migrations_paths` to `connection.assume_migrated_upto_version`.
+    ([Commit](https://github.com/rails/rails/commit/c1b14aded27e063ead32fa911aa53163d7cfc21a))
+
+*   Deprecate `ActiveRecord::Result#to_hash` in favor of `ActiveRecord::Result#to_a`.
+    ([Commit](https://github.com/rails/rails/commit/16510d609c601aa7d466809f3073ec3313e08937))
+
+*   Deprecate methods in `DatabaseLimits`: `column_name_length`, `table_name_length`,
+    `columns_per_table`, `indexes_per_table`, `columns_per_multicolumn_index`,
+    `sql_query_length`, and `joins_per_query`.
+    ([Commit](https://github.com/rails/rails/commit/e0a1235f7df0fa193c7e299a5adee88db246b44f))
+
+*   Deprecate `update_attributes`/`!` in favor of `update`/`!`.
+    ([Commit](https://github.com/rails/rails/commit/5645149d3a27054450bd1130ff5715504638a5f5))
+
 ### Notable changes
 
+*   Add `rails db:prepare` to migrate or setup a database by creates the database, loading the schema,
+    running the migrations, and adding the seed data.
+    ([Pull Request](https://github.com/rails/rails/pull/35768))
+
+*   Add `after_save_commit` callback as shortcut for `after_commit :hook, on: [ :create, :update ]`.
+    ([Pull Request](https://github.com/rails/rails/pull/35804))
+
+*   Add `ActiveRecord::Relation#extract_associated` for extracting associated records from a relation.
+    ([Pull Request](https://github.com/rails/rails/pull/35784))
+
+*   Add `ActiveRecord::Relation#annotate` for adding SQL comments to ActiveRecord::Relation queries.
+    ([Pull Request](https://github.com/rails/rails/pull/35617))
+
+*   Add support for setting Optimizer Hints on databases.
+    ([Pull Request](https://github.com/rails/rails/pull/35615))
+
+*   Add `insert_all`/`insert_all!`/`upsert_all` methods for doing bulk inserts.
+    ([Pull Request](https://github.com/rails/rails/pull/35631))
+
+*   Add `rails db:seed:replant` that truncates tables of each database
+    for ther current environment and loads the seeds.
+    ([Pull Request](https://github.com/rails/rails/pull/34779))
+
+*   Add `reselect` method, which is a short-hand for `unscope(:select).select(fields)`.
+    ([Pull Request](https://github.com/rails/rails/pull/33611))
+
+*   Add negative scopes for all enum values.
+    ([Pull Request](https://github.com/rails/rails/pull/35381))
+
+*   Add `#destroy_by` and `#delete_by` for conditional removals.
+    ([Pull Request](https://github.com/rails/rails/pull/35316))
+
+*   Add the ability to automatically switch database connections.
+    ([Pull Request](https://github.com/rails/rails/pull/35073))
+
+*   Add the ability to prevent writes to a database for the duration of a block.
+    ([Pull Request](https://github.com/rails/rails/pull/34505))
+
+*   Add an API for switching connections to support multiple databases. This
```

Comment:
Perhaps we can just keep one line. `Add an API for switching connections to support multiple databases.`